test/sequential/test-child-process-emfile.js

```diff
@@ -1,30 +1,46 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
-var spawn = require('child_process').spawn;
-var fs = require('fs');
+const common = require('../common');
+const assert = require('assert');
+const child_process = require('child_process');
+const fs = require('fs');
 
 if (common.isWindows) {
   console.log('1..0 # Skipped: no RLIMIT_NOFILE on Windows');
   return;
 }
 
-var openFds = [];
+const ulimit = Number(child_process.execSync('ulimit -n'));
```

Comment:
`ulimit -Hn` works fine on OS X without super-user privileges. It's setting (e.g., `ulimit -Hn 64`) that requires super-user.

Good catch on the `unlimited` string. I'll fix that...